aws-analytics-pinpoint/src/main/java/com/amplifyframework/analytics/pinpoint/AmazonPinpointAnalyticsPlugin.java

```diff
@@ -305,7 +305,17 @@ public String getPluginKey() {
      * {@inheritDoc}
      */
     @Override
-    public void configure(@NonNull JSONObject pluginConfiguration, @NonNull Context context) throws AnalyticsException {
+    public void configure(
+            @Nullable JSONObject pluginConfiguration,
+            @NonNull Context context
+    ) throws AnalyticsException {
+        if (pluginConfiguration == null) {
+            throw new AnalyticsException(
```

Comment:
Hm, I think this means the annotation should say `@NonNull`. If you throw an exception whenever you see a null param, it can't really be null.

(In Java, any object _can_ be null, but the annotation means what it _ought to_ be, for the software to work as designed.)